src/plugin/Plugin.cc

```diff
@@ -446,11 +461,19 @@ bool Plugin::HookReporter(const std::string& prefix, const EventHandlerPtr event
 	return true;
 	}
 
-void Plugin::MetaHookPre(HookType hook, const HookArgumentList& args)
+void Plugin::MetaHookPre(::plugin::HookType hook, const HookArgumentList& args)
+	{
+	}
+
+void Plugin::MetaHookPost(::plugin::HookType hook, const HookArgumentList& args, HookArgument result)
 	{
 	}
 
-void Plugin::MetaHookPost(HookType hook, const HookArgumentList& args, HookArgument result)
+void Plugin::MetaHookPre(::zeek::plugin::HookType hook, const HookArgumentList& args)
+	{
+	}
+
+void Plugin::MetaHookPost(::zeek::plugin::HookType hook, const HookArgumentList& args, HookArgument result)
 	{
 	}
```

Comment:
Shouldn't the default implementation of these new `MetaHook{Pre,Post}` methods call the old, deprecated method to remain backwards compatible during the the deprecation period?

Might also leave some plugin btest un-migrated as a way to verify backward compat.